### PR TITLE
bump kubectl-neat to v2.0.3

### PR DESCRIPTION
**Pull request by itaysk**
this adds arm support for linux and mac

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/neat.yaml` is not a straightforward version bump: change on line `3` does not seem like a version bump: [`-        os: darwin`]

File: plugins/neat.yaml [hunk 1]
```diff
@@ -19,18 +19,38 @@ spec:
...
-        arch: amd64
+        arch: arm64
         os: linux
-    sha256: 95dac6c3776fcf14bf19aa158b880d9df39f059354246673d18a8af232513925
-    uri: https://github.com/itaysk/kubectl-neat/releases/download/v2.0.2/kubectl-neat_linux.tar.gz
+    sha256: 1007ac8c87711f9c9af54df7bc2eebcd509dd4bb71bf2f4d93f5c21288a6908b
+    uri: https://github.com/itaysk/kubectl-neat/releases/download/v2.0.3/kubectl-neat_linux_arm64.tar.gz
+  - bin: ./kubectl-neat
+    files:
+    - from: /*
```
Comment (ahmetb) [inline plugins/neat.yaml:37]: This is redundant I believe. You can drop the entire "files" fields if you're copying out all files?

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/neat.yaml` is not a straightforward version bump: change on line `3` does not seem like a version bump: [`-    files:`]

File: plugins/neat.yaml (hunk 1 above)
Comment (itaysk) [reply plugins/neat.yaml]: thanks, I've remove it (in another commit, so probably squash before merging)

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/neat.yaml` is not a straightforward version bump: change on line `3` does not seem like a version bump: [`-    files:`]

**Pull request merged**